### DISCUSSION TITLE
Windows 11 support

### DISCUSSION THREAD

**Discussion by Gregory-Bbr** [jgraph/drawio-desktop#1840] [Q&A] [answered]
Hello,
Can you tell me or show me where to find information about Windows 11 support?
I mean from which version is Windows 11 supported?
Thanks & regards.

**Comment by davidjgraph** [comment]
honestly, no idea.

**Reply by Gregory-Bbr** [reply]
Ok, I'll test then, because we use for the moment the version 19.0.3. If that version is not fully compatible Windows 11, I have to update before our computers are updated to Windows 11.

**Reply by davidjgraph** [reply]
You're missing critical security fixes running 19.0.3. Why can you not update?

**Reply by davidjgraph** [reply, marked as answer]
But 19.0.3 is win 11 compatible. I tested versions before that on Win 11.

**Reply by Gregory-Bbr** [reply]
I don't know the updating process in our company. I'll have to check with colleagues how to do it. Thanks for the information.

**Reply by Gregory-Bbr** [reply]
Tested today, version 19.0.3 runs perfectly under Windows 11
